frappe/bench_interface.py

```diff
@@ -324,13 +326,18 @@ def __json__(self) -> dict[str, Any]:  # type: ignore[no-any-explicit,no-any-dec
 		return {naming.get(k, k): v for k, v in self.__dict__.items() if k not in excluded}  # type: ignore[no-any-expr]
 
 
-class Sites(_PathResolvable, ConfigHandler, _Serializable):
+class Sites(ConfigHandler, _PathResolvable, _Serializable):
 	SCOPE_ALL_SITES: Final = "__scope-all-sites__"
 	__sites: dict[str, Site]
 	__path_name = "sites"
 	__env_variable = "FRAPPE_SITES_PATH"
 
 	if legacy_env_path := os.environ.get("SITES_PATH"):
+		deprecation_warning(
+			"2024-12-06",
+			"v17",
+			"Specifying sites path via SITES_PATH env variable is deprecated, use the canonical FRAPPE_SITES_PATH, instead.",
+		)
 		os.environ["FRAPPE_SITES_PATH"] = legacy_env_path
```

Comment:
What does this mean? Do we still support some kind of path configuration?

Reply:
Yes, uniformly so as `FRAPPE_[classnameupper]_PATH`, for all runtime paths.